tests/app/views/test_drafts.py

```diff
@@ -1493,3 +1493,82 @@ def test_should_not_complete_invalid_dos_draft(self):
         data = json.loads(complete.get_data())
         assert_in("specialist_required", "{}".format(data['error']['_form']))
         assert_equal(complete.status_code, 400)
+
+
+class TestUpdateDraftStatus(BaseApplicationTest):
+    def setup(self):
+        super(TestUpdateDraftStatus, self).setup()
+
+        with self.app.app_context():
+            db.session.add(Supplier(supplier_id=1, name=u"Supplier 1"))
+            db.session.add(
+                ContactInformation(
+                    supplier_id=1,
+                    contact_name=u"Test",
+                    email=u"supplier@user.dmdev",
+                    postcode=u"SW1A 1AA"
+                )
+            )
+            Framework.query.filter_by(slug='g-cloud-7').update(dict(status='open'))
+            db.session.commit()
+        draft_json = self.load_example_listing("G7-SCS")
+        draft_json['frameworkSlug'] = 'g-cloud-7'
+        create_draft_json = {
+            'update_details': {
+                'updated_by': 'joeblogs'
+            },
+            'services': draft_json
+        }
+
+        draft = self.client.post(
+            '/draft-services',
+            data=json.dumps(create_draft_json),
+            content_type='application/json')
+
+        self.draft = json.loads(draft.get_data())['services']
+        self.draft_id = self.draft['id']
+
+    def test_update_draft_status(self):
+        res = self.client.post(
+            '/draft-services/%s/update-status' % self.draft_id,
```

Comment:
Why not use the `.format()` method for these?  Percentage formatting is less good, [apparently](https://docs.python.org/3/library/stdtypes.html#printf-style-string-formatting).